src/settings/Settings.tsx

```diff
@@ -461,6 +462,14 @@ export const SETTINGS: { [setting: string]: ISetting } = {
         description: _td("Under active development"),
         default: false,
     },
+    [Features.VoiceBroadcastForceSmallChunks]: {
+        isFeature: true,
```

Comment:
If this is a developer feature, I believe we should not promote that in labs. 
It should either go in the developer tools or maybe in `config.json` and not be promoted in the UI

Reply:
It should be available from the UI to allow testing on every deployment, where you eventually cannot easily change the config → moved to dev tools. Updated screenshot in description.